extension/src/test/e2e/extension.test.ts

```diff
@@ -142,7 +143,7 @@ describe('Plots Webview', function () {
     this.timeout(60000)
     const workbench = await browser.getWorkbench()
     await workbench.openCommandPrompt()
-    await browser.keys([...'DVC: Show Plots', 'ArrowDown', 'Enter'])
+    await browser.keys([...'DVC: Show Plots', Key.ArrowDown, Key.Enter])
```

Comment:
[F] I think this is the breaking change that we got pinned by before... the window dies without a helpful message when the `Key` enum is not used.